urwid/widget/wimp.py

```diff
@@ -142,12 +142,66 @@ def sizing(self):
     # (this variable is picked up by the MetaSignals metaclass)
     signals: typing.ClassVar[list[str]] = ["change", "postchange"]
 
+    @typing.overload
```

Comment:
decorated code is only type annotation explaining difference for useful argument types depending on `has_mixed` and `user_data`